docs/searchpredicates.txt

```diff
@@ -110,7 +110,14 @@ Geoshape.circle(37.97, 23.72, 50)
 Geoshape.box(37.97, 23.72, 38.97, 24.72)
 //WKT (requires JTS for polygon, linestring and multi-point/line/polygon geometries)
 Geoshape.fromWkt("POLYGON ((35.4 48.9, 35.6 48.9, 35.6 49.1, 35.4 49.1, 35.4 48.9))")
-
+//MultiPoint
+Geoshape.geoshape(Geoshape.getShapeFactory().multiPoint().pointXY(60.0, 60.0).pointXY(120.0, 60.0).build());
+//MultiLine
+Geoshape.geoshape(Geoshape.getShapeFactory().multiLineString().add(Geoshape.getShapeFactory().lineString().pointXY(59.0, 60.0).pointXY(61.0, 60.0)).add(Geoshape.getShapeFactory().lineString().pointXY(119.0, 60.0).pointXY(121.0, 60.0)).build());
+//MultiPolygon
+Geoshape.geoshape(Geoshape.getShapeFactory().multiPolygon().add(Geoshape.getShapeFactory().polygon().pointXY(59.0, 59.0).pointXY(61.0, 59.0).pointXY(61.0, 61.0).pointXY(59.0, 61.0).pointXY(59.0, 59.0)).add(Geoshape.getShapeFactory().polygon().pointXY(119.0, 59.0).pointXY(121.0, 59.0).pointXY(121.0, 61.0).pointXY(119.0, 61.0).pointXY(119.0, 59.0)).build());
+//GeometryCollection
+Geoshape.geoshape(Geoshape.getShapeFactory().multiShape(Shape.class).add(Geoshape.getShapeFactory().pointXY(60.0, 60.0)).add(Geoshape.getShapeFactory().lineString().pointXY(119.0, 60.0).pointXY(121.0, 60.0).build()).build());
```

Comment:
I create a Geoshape.getGeometryCollectionBuilder() like that you do not need to import org.locationtech.spatial4j.shape.Shape